test/dynamo/test_higher_order_ops.py

```diff
@@ -5262,9 +5262,9 @@ def forward(self, L_x_: "f32[4, 3]"):
         _set_fwd_grad_enabled_1 = torch._C._set_fwd_grad_enabled(True);  _set_fwd_grad_enabled_1 = None
         _jvp_decrement_nesting = torch._C._functorch._jvp_decrement_nesting();  _jvp_decrement_nesting = None
 
-        results: "f32[12, 4, 3]" = torch._C._functorch._remove_batch_dim(tangents_out_unflatten, 1, 12, 0);  tangents_out_unflatten = None
+        results: "f32[12, 4, 3]" = torch._functorch.vmap._remove_batch_dim_python_wrapper(tangents_out_unflatten, 1, 12, 0);  tangents_out_unflatten = None
 
-        _vmap_decrement_nesting = torch._C._functorch._vmap_decrement_nesting();  _vmap_decrement_nesting = None
+        _vmap_decrement_nesting_python_wrapper = torch._functorch.vmap._vmap_decrement_nesting_python_wrapper();  _vmap_decrement_nesting_python_wrapper = None
```

Comment:
I don't want to go down everything and name them "*_python_wrapper"

Reply:
I think the original argument was that we don't want to add tf handler to the C++ function. Are you saying we still define a python wrapper and inside there we just proxy torch._C._functorch._vmap_decrement_nesting?